plenum/server/consensus/ordering_service.py

```diff
@@ -2383,12 +2390,14 @@ def process_old_view_preprepare_reply(self, msg: OldViewPrePrepareReply, sender)
         for pp_dict in msg.preprepares:
             try:
                 pp = PrePrepare(**pp_dict)
```

Comment:
It looks like that we process all `OldViewPrePrepareReply` (from all nodes) regardless if the PrePrepare has been already ordered. That means that
1) PREPARE will be sent multiple times (that's why we see `PREPARE request already received` sometimes). Doesn't break anything, but just inconsistent a bit.
2) `_reordered_in_new_view` is called on every received `OldViewPrePrepareReply` which means that 
- we call `self._stasher.process_all_stashed(STASH_VIEW_3PC)` which may unstash more than needed if there are multiple view changes in a row. Should not break anything, but just inconsistent a bit.
- we send `ReOrderedInNewView` multiple times, which resets monitor.  Again, should not break anything, but just inconsistent a bit. A risk of false positive view changes.

Reply:
Fixed